tools/ci/download_bin.sh

```diff
@@ -44,11 +44,19 @@ download_artifact_linux()
     else
         link=$(echo ${response}| grep -o 'https://[^"]*' | grep 'mini')
     fi
+    if [ -z "${link}" ];then
+        build_num=$(( build_num - 1 ))
+        response="$(curl https://circleci.com/api/v1.1/project/github/${org}/${repo}/${build_num}/artifacts?circle-token= 2>/dev/null)"
+        if [ -z "${download_mini}" ];then
+            link=$(echo ${response}| grep -o 'https://[^"]*' | grep -v 'mini'| tail -n 1)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5667311)